Lanchonete/Lanchonete/ Features/Confirmation/Data/SandwichManager.swift

```diff
@@ -0,0 +1,80 @@
+//
+//  SandwichManager.swift
+//  Lanchonete
+//
+//  Created by vera.dias on 1/13/21.
+//
+
+import UIKit
```

Comment:
Is this class a "domain" one? If so, it shouldn't be necessary to import UIKit, since UIKit is a framework that belongs in the implementation details.

Reply:
I removed this can you take a look again? 😄